### PR TITLE
Modify &fight to make developers do twice as much damage

### PR DESCRIPTION
**Pull request by bkeeneykid**

### CODE REVIEW

**Review by tweirtx — APPROVED** [review]
lmao im down

**Comment by AidanFromProgramming** [comment]
Thats too good

**Review by tweirtx — APPROVED** [review]

File: dozer/cogs/fun.py [hunk 1]
```diff
@@ -36,6 +36,8 @@ async def fight(self, ctx, opponent: discord.Member):
         while hps[0] > 0 and hps[1] > 0:
             opp_idx = (turn + 1) % 2
             damage = random.choice(damages)
+            if players[turn].id in ctx.bot.config['developers'] or players[turn].id == ctx.me:
```
Comment (octocynth) [inline dozer/cogs/fun.py:39]: Should this be `players[turn] == ctx.me` or `players[turn].id == ctx.me.id`?

**Review by octocynth — CHANGES_REQUESTED** [review]
😂 One little fix, and then I'll look forward to seeing this in action!

**Review by octocynth — APPROVED** [review]

**Review by tweirtx — APPROVED** [review]

**Pull request merged**